item-spec/item-spec.md

```diff
@@ -167,12 +167,23 @@ or streamed. It is allowed to add additional fields.
 | href        | string    | **REQUIRED.** Link to the asset object. Relative and absolute links are both allowed. |
 | title       | string    | The displayed title for clients and users. |
 | description | string    | A description of the Asset providing additional details, such as how it was processed or created. [CommonMark 0.29](http://commonmark.org/) syntax MAY be used for rich text representation. |
-| type        | string    | [Media type](#media-types) of the asset. |
+| type        | string    | [Media type](#asset-media-type) of the asset. See the [common media types](../best-practices.md#common-media-types-in-stac) in the best practice doc for commonly used asset types. |
 | roles       | \[string] | The [semantic roles](#asset-role-types) of the asset, similar to the use of `rel` in links. |
 
 Beyond to the fields above, any of the [additional fields](#additional-fields) *may* be added to the assets. But this
 is recommended only in special cases, see [Additional Fields for Assets](#additional-fields-for-assets)) for more information.
 
+#### Asset Media Type
+
+Any media type can be used in an Item's asset `type` field, and [registered](https://www.iana.org/assignments/media-types/media-types.xhtml) 
+Media Types are preferred. In cases where custom vendor-specific media types are necessary, they should
```

Comment:
I think the .vnd info is a bit out of scope here. Most people will not make up their own media types and register them.

Reply:
Good catch, will move this out / link to there.